sdk/elasticsan/Azure.ResourceManager.ElasticSan/src/Generated/Models/ElasticSanSkuName.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.ResourceManager.ElasticSan.Models
+{
+    /// <summary> The sku name. </summary>
+    public readonly partial struct ElasticSanSkuName : IEquatable<ElasticSanSkuName>
+    {
+        private readonly string _value;
+
+        /// <summary> Initializes a new instance of <see cref="ElasticSanSkuName"/>. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public ElasticSanSkuName(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string StandardLRSValue = "Standard_LRS";
+        private const string PremiumLRSValue = "Premium_LRS";
+        private const string StandardZRSValue = "Standard_ZRS";
+
+        /// <summary> Standard locally redundant storage. </summary>
+        public static ElasticSanSkuName StandardLRS { get; } = new ElasticSanSkuName(StandardLRSValue);
```

Comment:
Although this is an enum value name, I'm not sure if we should change to `StandardLrs`?